apps/etterlatte-behandling/src/main/kotlin/common/klienter/PdlKlient.kt

```diff
@@ -40,6 +43,21 @@ class PdlKlientImpl(
         }
         return response
     }
+
+    override fun hentGeografiskTilknytning(foedselsnummer: String): GeografiskTilknytning {
+        val fnr = Folkeregisteridentifikator.of(foedselsnummer)
+        logger.info("Henter geografisk tilknytning for $fnr")
```

Comment:
Denne bør vi nok rydde bort 😉

Reply:
Har vi ikke lov å logge maskerte ? Folkeregisteridentifikator maskere siste x siffer

Reply:
Hmm, godt spørsmål. Vi har vel sikkert egentlig *lov* til det, men trengs det?

Reply:
Tja - sikkert ikke. Var basert på hentPdlModell men der logger de rolle.

Kan fjerne.

Men vil fortsatt ha svar på spørsmål i slack for fremtiden.